<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PointInRootSetterWindow</class>
 <widget class="QWidget" name="PointInRootSetterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>176</width>
    <height>278</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PointInRootSetterWindow</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="positionLabel">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QRadioButton" name="fieldButton_regular">
           <property name="text">
            <string>Regular field</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="fieldButton_ropeField">
           <property name="text">
            <string>Rope field</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="fieldButton_ladderUp">
           <property name="text">
            <string>Lader-up</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="fieldButton_ladderDown">
           <property name="text">
            <string>Lader-down</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="fieldButton_stairsUp">
           <property name="text">
            <string>Stairs-up</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="fieldButton_stairsDown">
           <property name="text">
            <string>Stairs-Down</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="fieldButton_openHole">
           <property name="text">
            <string>Hole-always oepn</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="fieldButton_holeShovelNeeded">
           <property name="text">
            <string>Hole-need shovel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="fieldButton_teleport">
           <property name="text">
            <string>Teleport</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="okButton">
           <property name="text">
            <string>Ok</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cancelButton">
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>PointInRootSetterWindow</receiver>
   <slot>okButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>257</y>
    </hint>
    <hint type="destinationlabel">
     <x>89</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>PointInRootSetterWindow</receiver>
   <slot>cancelButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>257</y>
    </hint>
    <hint type="destinationlabel">
     <x>89</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>cancelButtonClicked()</slot>
  <slot>okButtonClicked()</slot>
 </slots>
</ui>
